### PR TITLE
DOCS-1401: Update README to match template

### PR DESCRIPTION
**Pull request by andf-viam**
Update README to match [template](https://github.com/viam-labs/generator-viam-module/blob/main/generators/app/templates/python/README.md).
- This module was originally intended to function as a learning resource for new module developers, so I have retained the existing extended usage and forking guidance in a new, penultimate `Learning resources` section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,88 @@
...
 This module includes a `setup.sh` file to help configure a `virtualenv` for this module.
 Follow the instructions to [prepare your Python virtual environment](https://docs.viam.com/build/program/python-venv/) to learn how to set up and distribute a `virtualenv` configuration with your custom module.
 
-### Use CI to automatically publish on release
+#### Use CI to automatically publish on release
 
 This module includes a [`.github/workflows`](https://github.com/viam-labs/python-example-module/tree/main/.github/workflows) directory, which contains workflows that you can use to automatically publish a new version of your module to the Viam registry when you create a GitHub release.
 For more information, see [Update a module using a GitHub action](https://docs.viam.com/registry/upload/#update-an-existing-module-using-a-github-action).
 
-## Next Steps
```
Comment (npentrel) [inline README.md:48]: I think this is good info? Why remove

Comment (npentrel) [reply README.md:48]: ad least the first one with testing is helpful

**Review by npentrel — APPROVED** [review]
LGTM % putting the testing info back in

File: README.md (hunk 1 above)
Comment (andf-viam) [reply README.md:48]: I moved Next steps up to line 56; the first sentence directs users to the Data tab to "test" (verify) that data has indeed been captured. Is that what you meant?

Comment (npentrel) [reply README.md:48]: Yep sorry

**Pull request merged**